withastro/astro
# docs: spanish translation

## Changes

- Spanish translation
- In the pending translation files, loads the English version and include a heading note.

- ToDo:
   - reference/renderer-reference
   - guides/markdown-content
   - guides/pagination
   - guides/publish-to-npm
   - guides/styling

## Testing



- Check /es/* pages to check the content is in spanish.
- Check pending translate files show content in ingles with a heading note.


## Docs



Just Spanish translation of the English version.
